### PR TITLE
chore: add sample for combined project with Teams Tab and Outlook add-in

### PR DESCRIPTION
**Pull request by 1yefuwang1**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_sprints/taskboard/Teams%20Extensibility%20E2E%20Team/Microsoft%20Teams%20Extensibility/CY23-2.2?workitem=17278839

### CODE REVIEW

File: packages/fx-core/src/common/samples-config-v3.json [hunk 1]
```diff
@@ -185,6 +185,16 @@
             "time": "5min to run",
             "configuration": "Ready for debug",
             "suggested": false
+        },
+        {
+          "id": "hello-world-teams-tab-and-outlook-add-in",
+          "title": "Hello World Teams tab and Outlook add-in",
```
Comment (tecton) [inline packages/fx-core/src/common/samples-config-v3.json:191]: Better to follow the title-style capitalization rule:
```suggestion
          "title": "Hello World Teams Tab and Outlook Add-in",
```

File: packages/fx-core/src/common/samples-config-v3.json [hunk 2]
```diff
@@ -185,6 +185,16 @@
...
+          "shortDescription": "A hello world project that contains both Teams tab and Outlook Add-in capability",
+          "fullDescription": "A hello world project that contains both Teams tab and Outlook Add-in capability",
```
Comment (tecton) [inline packages/fx-core/src/common/samples-config-v3.json:193]: Does this need PM's assistant to add full description?

Comment (1yefuwang1) [reply packages/fx-core/src/common/samples-config-v3.json:193]: @MuyangAmigo do we need a full description?

**Comment by HuihuiWu-Microsoft** [comment]
Suggested size for the screenshot/gif is 1600*920/800*460. If you have it with other sizes, please keep same image ratio according  to https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_wiki/wikis/Microsoft-Teams-Extensibility.wiki/388807/Onboard-Sample-Gallery

File: packages/fx-core/src/common/samples-config-v3.json (hunk 2 above)
Comment (MuyangAmigo) [reply packages/fx-core/src/common/samples-config-v3.json:193]: Not really.

File: packages/fx-core/src/common/samples-config-v3.json (hunk 1 above)
Comment (1yefuwang1) [reply packages/fx-core/src/common/samples-config-v3.json]: Changed "tab" to "Tab". Outlook add-in with lower-case "a" is the term to use in all docs.

**Review by HuihuiWu-Microsoft — DISMISSED** [review]

**Review by tecton — APPROVED** [review]

**Pull request merged**